README.md

```diff
@@ -1,2 +1,16 @@
 # awesome-embodied-ai-ros
-A curated list of repositories and papers on Embodied AI in ROS ecosystem.
+
+[![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+<img src="https://img.shields.io/badge/Contributions-Welcome-278ea5" alt="Contrib"/>
+
+A curated list of projects and papers on Embodied AI in ROS ecosystem.
+
+## Projects
+
```

Comment:
Please order alphabetically and perhaps also divide into frameworks (RAI / ROSA), model integrations (llama_ros), user tools (ros2ai), and projects (tabletop-handybot)

Reply:
the division can be done later on